tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/CreateOrAlterProcedureVisitor.cs

```diff
@@ -248,6 +255,50 @@ private MemberDeclarationSyntax AddPopulateCommandMethodForTableValuedParameters
                                     .WithNameColon(NameColon(ParameterNameForParameter(p)))).ToArray())));
     }
 
+    private MethodDeclarationSyntax[] AddGetOutputMethod(CreateOrAlterProcedureStatement node)
+    {
+        List<TypeSyntax> outputTypes = new List<TypeSyntax>();
+        List<ExpressionSyntax> outputExpressions = new List<ExpressionSyntax>();
+        IEnumerable<ProcedureParameter> procedureParameters = node.Parameters.Where(p => p.Modifier == ParameterModifier.Output);
+
+        if (procedureParameters.Count() == 0)
+        {
+            return new MethodDeclarationSyntax[0];
+        }
+
+        foreach (ProcedureParameter parameter in procedureParameters)
+        {
+            outputTypes.Add(DataTypeReferenceToClrType(parameter.DataType, parameter.Value != null));
+            outputExpressions.Add(InvocationExpression(
+                                    MemberAccessExpression(
```

Comment:
nit: no need to align these arbitrarily based on the length of identifiers. Simply indent it once per level:
```csharp
outputExpressions.Add(
    InvocationExpression(
        MemberAccessExpression(
            SyntaxKind.SimpleMemberAccessExpression,
            IdentifierName(FieldNameForParameter(parameter)),
            IdentifierName("GetOutputValue")))
    .AddArgumentListArguments(
        Argument(IdentifierName(CommandParameterName))));
```

Reply:
Thanks, updated.